src/views/Stake/index.tsx

```diff
@@ -33,6 +37,22 @@ export default function Stake() {
     setMevMax(0)
     setBlsKey('')
     setWizard(null)
+	bribe = isValidatorIncentivized(value)
+  }
+
+  const isValidatorIncentivized = (value: string) => {
+    try {
+	  const bribeVaultAddresses = makeFrenDelegationBribeVaultAddress()
+	  const bribeVaultAddress = bribeVaultAddresses[0] // TO-DO: iterate each BribeVault and show rewards from each?
```

Comment:
@vdallco

Reply:
This TO-DO is needed if we want to pull in bribe data from more than one BribeVault per chain. This isn't necessary but the comment could be useful in the future if we decide to implement this behavior. I think it's fine to approve, but I'll leave that up to @stakehouse-dev 

Thanks

Reply:
@stakehouse-dev all set. Pushed new commits to:
- Update lsd-wizard dependency to 3.5.0 in package.json and package-lock.json
- Append "per 1 ETH" to the "Earn up to" label for incentivized validators (as the tokenToETHRatio is displayed)

Reply:
@vdallco we have a build issue - can you open another PR with the fix? issue:
```
4:00:17 PM: ./src/views/Stake/index.tsx
4:00:17 PM: SyntaxError: /opt/build/repo/src/views/Stake/index.tsx: Unexpected token (53:4)
4:00:17 PM:   51 |       const bribeData = bribeWizard.utils.getFrenDelegationBribesByBLS(value)
4:00:17 PM:   52 | 	  return bribeData
4:00:17 PM: > 53 |     catch(err: any){
4:00:17 PM:      |     ^
4:00:17 PM:   54 |       return false
4:00:17 PM:   55 |     }
4:00:17 PM:   56 |   }
```

Reply:
fixed here: https://github.com/stakehouse-dev/fren-delegation/blob/main/src/views/Stake/index.tsx

Reply:
Hey - can you explain how the vault contract checks that the validator is eligible for incentives? we are trying to check that this works and we wanted to know how you are adding eligible BLS keys so that it is surfaced in the UI. Alternatively, if you can give us BLS keys that are supposed to be eligible then we will be happy to test

Reply:
Hello, the vault contract is checking if the BLS key exists in the "deposits" mapping. Please see the recent changes in lsd-wizard to understand how this works. I've verified that lsd-wizard is pulling the correct data from the VaultContract in goerli. You can verify the same with this code (just enter a private-key):

`
import {Wizard} from './Blockswap-LSDWizard/config.mjs'; //'@blockswaplab/lsd-wizard';
import { ethers } from 'ethers';
import { InfuraProvider } from '@ethersproject/providers';

async function main() {
	const provider = new InfuraProvider("goerli", {
		projectId: '54bf0ea8b8764cc59081cf17457810df',
		projectSecret: 'bb783ad6e72c485fad1ea1e6802497e1'
	});

	const private_key = '';
    const account = new ethers.Wallet(private_key, provider);

    const blsPublicKey = '0xa86c8fd31cae571b883f1a6602e02d7506926f3a5c04ea5af751e0e1e61490da61f6ca2fe3847f32d6306f34900af856'; 

	console.log("Initializing lsd-wizard SDK")
    const wizard = new Wizard({signer: account, frenDelegationBribeVaultAddress:'0xeC01dfbE79412b078521793df4415AD8b84EDB27'})
	
    var bribeData = await wizard.utils.getFrenDelegationBribesByBLS(blsPublicKey);
	console.log("Bribe data: ");
	console.log(bribeData);
}

main()
    .then(() => process.exit(0))
    .catch(error => {
        console.error(error);
        process.exit(1);
    });
`

But when I enter 0xa86c8fd31cae571b883f1a6602e02d7506926f3a5c04ea5af751e0e1e61490da61f6ca2fe3847f32d6306f34900af856 into the Goerli Fren Delegation website, I don't see the new box show up for incentives.

Reply:
@stakehouse-dev

Reply:
thank you for the reply - we will look into this

Reply:
We have pointed the error to:
```
Error TypeError: Cannot read properties of undefined (reading 'getFrenDelegationBribesByBLS')
    at isValidatorIncentivized (index.tsx:54:1)
```

Reply:
@stakehouse-dev thank you for pointing that out. I have pushed a new PR to resolve this. Please see https://github.com/stakehouse-dev/fren-delegation/pull/6/

Reply:
@vdallco thanks. we directly copied the changes and brought it into main. there was an issue with utils package not initialising due to not having a liquid staking manager value - we'll fix the sdk but in the mean time something is now appearing

Reply:
https://main--fren-delegation.netlify.app/

Reply:
can you try this with the above bls key

Reply:
@stakehouse-dev Can you please show me where the setBribeState function is defined? I searched the entire org on GitHub but only found the references in ValidatorDetails.tsx

```
if (bribeData && bribeData.id && bribeData.tokenDecimals) {
    setBribeState(bribeData)
  }
```

Reply:
@stakehouse-dev Ignore my previous comment about setBribeState (now deleted). I found an issue preventing the number of tokens per ETH from displaying, and which causes the incentives box to show for every BLS regardless of bribe status. We needed to add .hex to tokenToEthRatio references. Please see that fix here: https://github.com/stakehouse-dev/fren-delegation/pull/8

I'm not sure why the token name doesn't appear. Perhaps the state isn't being set correctly. We could add debug logging to find out.